app/views/directives/key-value-editor.html

```diff
@@ -4,14 +4,10 @@
     ng-if="showHeader"
     class="key-value-editor-entry key-value-editor-entry-header">
     <div class="form-group key-value-editor-header key-header">
```

Comment:
Do we still need `form-group`?

Reply:
well, it's setting the margin-bottom but that could be set using the specific class *-header

If so we'd need to do the same for the KVE ones above to be consistent